code/modules/projectiles/gun_attachables.dm

```diff
@@ -1409,15 +1421,15 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
 	if(!isgun(attaching_to))
 		return FALSE
 	var/obj/item/weapon/gun/attaching_gun = attaching_to
-	if(CHECK_BITFIELD(attaching_gun.flags_gun_features, GUN_IS_SENTRY))
+	if(istype(master_gun, /obj/item/weapon/gun/sentry))
 		to_chat(attacher, span_warning("[attaching_gun] is already a sentry!"))
```

Comment:
why are you removing flags

Reply:
hyper told me to do this and the one below